index.html

```diff
@@ -4,19 +4,16 @@
 
 <nav class="main-menu"> 
     <div class="row">
-        <a href="./colleges/">
+        <a href="./colleges/francisco-goya/">
```

Comment:
This part is hardcoded for now. Will need to be parametrized (via configuration file) later